plugins/destination/duckdb/client/client.go

```diff
@@ -73,6 +73,14 @@ func (c *Client) Metrics() destination.Metrics {
 }
 
 func (c *Client) exec(ctx context.Context, query string, args ...any) error {
-	_, err := c.db.ExecContext(ctx, query, args...)
+	r, err := c.db.ExecContext(ctx, query, args...)
+	if c.spec.Version == "development" {
```

Comment:
Per a better way to communicate between the test framework and the plugin. Also only temporarily